malwarecage/web/src/components/QuickQuery.js

```diff
@@ -0,0 +1,129 @@
+import React, {Component} from 'react';
+
+import { ConfirmationModal } from "@malwarefront/ui";
+import api from "@malwarefront/api";
+
+import QuickQueryAddModal from "./QuickQueryAddModal";
+
+
+class QuickQuery extends Component {
+    state = {
+        queries: [],
+        quickQueryId: null,
+        isAddModalOpen: false,
+        isDeleteModalOpen: false,
+        modalError: ""
+    }
+
+    getQuickQueries = async () => {
+        try {
+            let response = await api.getQuickQueries(this.props.type)
+            this.setState({queries: response.data})
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+    componentDidMount() {
+        this.getQuickQueries()
+    }
+
+    submitQuickQuery = async (name) => {
+        if (!this.props.query || this.props.queryError !== false) {
+            this.setState({modalError: "You can't save empty or broken query. Search with it first."})
+        } else {
+            try {
+                await api.addQuickQuery(
+                    this.props.type,
+                    name,
+                    this.props.query
+                    )
+                this.getQuickQueries()
+                this.setState(
+                    {
+                        isAddModalOpen: false,
+                        modalError:""
+                    })
+            } catch (error) {
+                this.setState({modalError: error})
+            }
+        }
+    }
+
+    deleteQuery = async (id) => {
+        await api.deleteQuickQuery(id)
+        this.setState({quickQueryId: null, isDeleteModalOpen: false})
+        this.getQuickQueries()
+    }
+
+    handleError = (error) => {
+        this.setState({modalError: error});
+    }
+
+    render() {
+        let queryBadges = Object(this.state.queries).sort((a, b) => a.id - b.id).map((v) =>
+                <span key={v.id} className="badge bg-dark" style={{"cursor": "pointer", "color":"white"}}>
```

Comment:
Wrong style is used, use `badge-<colorclass>` instead of `bg-<colorclass>`
```suggestion
                <span key={v.id} className="badge badge-dark" style={{cursor: "pointer", color: "white"}}>
```